sema4ai/vscode-client/src/robo/actionPackage.ts

```diff
@@ -659,3 +659,33 @@ export const publishActionPackage = async () => {
         }
     );
 };
+
+export const buildActionPackage = async() => {
+    const workspaceDir = await askForWs();
```

Comment:
We need to make sure that this is valid. 
If not, we should give the user a helpful message on what they need to do.

Commands in VSCode can be run at any point in time without having a valid Workspace or project, or etc...

Reply:
Here also the function it self handles all the guidance for the user.